src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
```

Comment:
i would rename this to `MAX_BALLL_NUMBER`, since `BOUND` is not that informative